.azure/app-cloud-e2e.yml

```diff
@@ -153,3 +156,97 @@ jobs:
         # GRID_URL: $(LIGHTNING_CLOUD_URL)
         # _GRID_USERNAME: $(LIGHTNING_USERNAME)
       displayName: 'Clean Previous Apps'
+
+  - job: App_quick_start_e2e_testing
```

Comment:
This looks very the same as other e2e tests how does it differs that we can't add this just as another item in the tested examples?

Reply:
differences:
- longer timeout 1400 instead of 900 seconds
- more ram
- no pip cache 

I

Reply:
> * longer timeout 1400 instead of 900 seconds

That can be for all, I guess

> * more ram

also can be for all as it runs on the same machine

> * no pip cache

can be conditional, so skip for this case

Reply:
no problem, we can have it on the same job 👍

Reply:
updated